### PR TITLE
Modularize common.rs structs

### PR DESCRIPTION
**Pull request by gillespiecd**
Refactor each of the structs in `common.rs` into their own modules and tests. 

I think this will help modularize the code for future additions and make it clearer to read. I'm still learning how to reason about Rust modules and project structures, but feel like this is an improvement.

#### ✔ Checklist:

- [X] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [X] Tests for the changes have been added (for bug fixes / features);
- [ ] Docs have been added / updated (for bug fixes / features).

### CODE REVIEW

File: src/common.rs [hunk 1]
```diff
@@ -1,507 +1,9 @@
...
-        write!(
-            f,
-            "{}:{} {}: {}",
-            self.line.file, self.line.number, self.check_name, self.message
-        )
-    }
-}
+pub use crate::common::file_entry::FileEntry;
+pub use crate::common::line_entry::LineEntry;
+pub use crate::common::warning::Warning;
```
Comment (mgrachev) [inline src/common.rs:7]:
```suggestion
pub use file_entry::FileEntry;
pub use line_entry::LineEntry;
pub use warning::Warning;
```

Comment (gillespiecd) [reply src/common.rs]: Updated, that looks better 👍

**Comment by mgrachev** [comment]
> - [ ] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list)

Also, please do it 🙏

**Comment by gillespiecd** [comment]
@mgrachev Yes, of course, sorry about that! I added to the changelog.

**Comment by mgrachev** [comment]
Thank you! ✌️

**Pull request merged**